machine_types.go

```diff
@@ -754,6 +757,18 @@ type MachineSecret struct {
 	Name string `json:"name"`
 }
 
+// @description EnvVar defines an environment variable to be populated from a machine field, env_var
+// and field_ref are required.
+type EnvFrom struct {
+	// EnvVar is required and is the name of the environment variable that will be set from the
+	// secret. It must be a valid environment variable name.
+	EnvVar string `json:"env_var"`
+
+	// FieldRef selects a field of the Machine: supports id, version, app_name, `metadata.<KEY>`,
+	// private_ip, region, image.
+	FieldRef string `json:"field_ref"`
```

Comment:
can we make this an enum?

Reply:
> can we make this an enum?
Because this is a public API and not a Go specific one, I've been avoiding doing so because it's unclear how this would look for people using it from non-Go languages.

Reply:
Thanks for the link (https://github.com/swaggo/swag?tab=readme-ov-file#add-a-description-for-enum-items) on slack! Updated in https://github.com/superfly/fly-go/pull/12/commits/9b36df0ac6002960f40263c4c5b02b511655d288